src/CSharpTest.Net.CollectionsTest/ExampleCode.cs

```diff
@@ -65,7 +65,7 @@ public void LurchTableDemo()
             Assert.AreEqual(counts.Queued, counts.Dequeued);
         }
 
-        [Test]
+        //[Test]
```

Comment:
Why is the commented out?

Reply:
These tests don't seem to run locally with a clean pull of the main branch.
Reverted.